superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -69,15 +68,24 @@ type MenuChild = {
   usesRouter?: boolean;
 };
 
+export enum ButtonStyleTypes {
+  Primary = 'primary',
+  Secondary = 'secondary',
+  Dashed = 'dashed',
+  Danger = 'danger',
+  Link = 'link',
+  Warning = 'warning',
+  Success = 'success',
+}
+
+interface ButtonProps {
+  name: any;
+  onClick: OnClickHandler;
+  style: ButtonStyleTypes;
```

Comment:
NOT, but it might actually be good to call it `buttonStyle` instead of `style`, lest people think it's something CSS related. Also, further down in the code, you would then have `buttonStyle={btn.buttonStyle}` if that feels more aligned at all :)